google/cloud/spanner_v1/batch.py

```diff
@@ -70,6 +73,10 @@ def insert(self, table, columns, values):
         :param values: Values to be modified.
         """
         self._mutations.append(Mutation(insert=_make_write_pb(table, columns, values)))
+        add_event_on_current_span(
```

Comment:
That's a great question but it won't be a problem because:
* spans have  a default event limit of 128 https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#span-limits
* storage costs not really as pricing on telemetry data is by span count rather than span data count unless perhps they have their own storage system but that's unlikely for those who'd be storing tens of thousands of span events

What we can do is perhaps document how to change span event limits and also document that we add events for span mutations.